formbricks/formbricks
# feat(github action): Add's a new GitHub Action to deploy webapp to ECS cluster

# What does this PR accomplish?
This pull request introduces the following changes:

- Docker Image Creation: The GitHub Action is triggered upon pushes to the main branch. It initiates the creation of a Docker Image based on the latest code changes. The resulting Docker Image is now stored in the GitHub Package Repository under the name "formbricks-experimental."

- Dockerfile Modifications: The Dockerfile has been updated to incorporate support for Sentry, enhancing observability. Importantly, this modification is optional and does not impact the self-hosted Docker setup.

# How to test this PR?
To ensure the successful implementation of these changes, follow the testing steps below:
- Self-Test on Fork: https://github.com/sp6370/formbricks.
- Create Test PR: Generate a test pull request, and confirm that the "formbricks-experimental" image is successfully created. The GitHub Action should only conclude when the ECS cluster stabilizes. Additionally, verify that [ecs.formbricks.com/health](https://chat.openai.com/c/ecs.formbricks.com/health) indicates that everything is functioning correctly.

### Required


### Appreciated
